free-2.md

```diff
@@ -11,41 +11,118 @@ _What need is met by your idea? Who are you helping? What is that person's
+complex email filtration system.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Filter out unwanted email types.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would write a configuration file. This file would contain something
+like
+
+```
+name: "ISO war"
+keywords: ["ISO"]
+start-frequency: 60
+start-count: 3
+step-frequency: 3600
+to: "ISO"
+```
+
+This means, filter out emails that are considered to be "ISO war" emails. These
+emails contain the word "ISO". 
+Filter them out if more than 3 such emails are 
+sent over the course of 60 seconds and continue filtering them out until the 
+frequency drops below 1 email per 3600 seconds. 
+These filtered emails should go to a folder labeled "ISO".
+
+A configuration file using JSON or XML-like syntax would give the system a good
+level of customization, provided that the language has enough keys.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The user would run a command, maybe in the command line, to start the program.
+The program reads the configuration file and continuously reads incoming emails.
+It would filter emails that should be filtered at all times and it would
+filter emails once certain criteria (such as emails per second) are 
+triggered.
+
+Syntax errors could occur if users try to use a key that the program does not
+recognize. This error should be displayed after the user enters the command. 
+
+There could be a lot of silent errors where the program misinterprets the user's
+input and simply fails to filter out the specified emails or filters out emails
+the user wants to receive. It would be on the user to identify these errors.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to identify types of emails based on keywords and regular
+expressions.
+
+It should be possible to identify emails that require a frequency threshold.
+
+It should be impossible to do anything outside of filtering out emails,
+such as using loops.
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+The [Gmail API](https://developers.google.com/gmail/api/guides/filter_settings)
+has XML files that do a similar thing. 
+The API has good fluency and descriptive variable names.
+But since the syntax is based on Gmail's search syntax, it cannot take into 
+account factors such as email frequency.
```

Comment:
Also I assume the GMAIL API only applies to gmail accounts whereas your DSL could theoretically apply to many.